PacketDotNet/TcpPacket.cs

```diff
@@ -294,12 +300,36 @@ virtual public bool CWR
             set { setFlag(value, TcpFields.TCP_CWR_MASK); }
         }
 
+        virtual public bool NS
+        {
+            get { return (AllFlags & TcpFields.TCP_NS_MASK) != 0; }
+            set { setFlag(value, TcpFields.TCP_NS_MASK); }
+        }
+
+        virtual public bool FlagA

```

Comment:
I'm asking because looking at the docs on wikipedia it isn't clear which of these properties is the highest bit vs. the lowest bit in the reserved flags.